aws.yaml

```diff
@@ -199,6 +199,44 @@
     - name: cluster-operator
       provider: aws
       version: 0.21.4
+- version: 9.0.1
+  active: true
+  date: 2020-03-09T19:00:00Z
+  apps:
+    - app: cert-exporter
+      version: 1.2.1
+    - app: chart-operator
+      version: 0.11.5
+    - app: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.3
+    - app: coredns
+      componentVersion: 1.6.5
+      version: 1.1.3
+    - app: kube-state-metrics
+      componentVersion: 1.9.2
+      version: 1.0.2
+    - app: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - app: net-exporter
+      version: 1.6.0
+    - app: nginx-ingress-controller
+      componentVersion: 0.30.0
+      version: 1.6.0
+    - app: node-exporter
+      componentVersion: 0.18.1
+      version: 1.2.0
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 5.5.0
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      provider: aws
+      version: 0.23.4
 - version: 9.0.0
   active: true
```

Comment:
Previous release with same k8s minor version has to be deactivated.

Reply:
Customers are using this 9.0.0 release currently. If we deactivated it at the same time, maybe we give confusion to our customers?

Reply:
Nope, this is common thing. See https://github.com/giantswarm/giantswarm/issues/9146

Reply:
Fixed to false. PTAL.